Releases.md

```diff
@@ -0,0 +1,84 @@
+* [Introduction](#introduction)
+* [Versioning](#versioning)
+* [Tagging repositories](#tagging-repositories)
+* [Components](#components)
+* [Release checklist](#release-checklist)
+* [Release process](#release-process)
+
+## Introduction
+
+This document provides details about Kata Containers releases.
+
+## Versioning
+
+The Kata Containers project uses [semantic versioning](http://semver.org/) for all releases. Semantic versions are comprised of three fields in the form:
+
+```
+MAJOR.MINOR.PATCH
+```
+
+For examples: `1.0.0`, `1.0.0-rc.5`, and `99.123.77+foo.bar.baz.5`.
+
+Semantic versioning is used since the version number is able to convey clear information about how a new version relates to the previous version. It can also provide assurances to allow users to know for example when they must upgrade compared with when they may wish to upgrade:
```

Comment:
Suggested rewrite of second sentence: ... For example, semantic versioning can also provide assurances to allow users to know when they must upgrade compared with when they might want to upgrade: